packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/ModelConfigurationToTables.java

```diff
@@ -13,9 +12,92 @@
 import software.uncharted.terarium.hmiserver.models.dataservice.modelparts.ModelParameter;
 import software.uncharted.terarium.hmiserver.models.dataservice.modelparts.semantics.State;
 
-public class ModelConfigurationLatexTable {
+public class ModelConfigurationToTables {
 
-	public static String generateLatexTable(Model model, ModelConfiguration modelConfiguration) {
+	public static String generateCsv(Model model, ModelConfiguration modelConfiguration) {
+		StringBuilder csv = new StringBuilder();
+
+		// Add the header
+		csv.append("ID,Name,Value,Units,Description,Source\n");
```

Comment:
Are we sure we want these columns across both initials and parameter? because it isn't obvious how to tell them apart, for example:

```
S, S, 1.0, null, null, null
beta, beta, 0.02, null, null, null
```

One is initial and one is parameter. Should we have an additional `type` column to differentiate?

Reply:
Sure, if necessary